internals/cli/cmd_check_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2025 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package cli_test
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"gopkg.in/check.v1"
+
+	"github.com/canonical/pebble/internals/cli"
+)
+
+func (s *PebbleSuite) TestCheck(c *check.C) {
+	s.RedirectClientToTestServer(func(w http.ResponseWriter, r *http.Request) {
+		c.Assert(r.Method, check.Equals, "GET")
+		c.Assert(r.URL.Path, check.Equals, "/v1/checks")
+		c.Assert(r.URL.Query(), check.DeepEquals, url.Values{"names": {"chk1"}})
+		fmt.Fprint(w, `
+{
+    "type": "sync",
+    "status-code": 200,
+    "result": [{"name": "chk1", "startup": "enabled", "status": "up", "threshold": 3, "change-id": "1"}]
+}`)
+	})
+	rest, err := cli.ParserForTest().ParseArgs([]string{"check", "chk1"})
+	c.Assert(err, check.IsNil)
+	c.Assert(rest, check.HasLen, 0)
+	c.Check(s.Stdout(), check.Equals, `
+name: chk1
+level: ""
+startup: enabled
+status: up
+failures: 0
+threshold: 3
+changeid: "1"
```

Comment:
Notice how the change ID is coming through. Should be `change-id`. Per my previous example that I wrote in the spec, I think this should be:

```
name: chk1
level: alive      # omitted if empty
startup: enabled
status: up
failures: 0       # included if 0
threshold: 3
change-id: "42"   # omitted if empty
```

So level and change-id would be marked `omitempty`. I think it's good to keep "failures" without omitempty, so that the user can clearly see there were 0 failures if so.